app/src/main/java/com/quran/labs/androidquran/QuranDataActivity.java

```diff
@@ -341,30 +342,6 @@ protected Boolean doInBackground(Void... params) {
             .subscribe();
       }
 
-      // in 2.9.0, there was an initial release of the new madani images - since then, we
-      // moved to a new set of images that let us draw the ayah markers and sura headers
-      // (and are, consequently, much lighter). they also properly center in the pages,
-      // unlike the older ones. the number of people who get this update are very few, and
-      // so temporarily add some code to remove this directory for now. this code can be
-      // removed in a future version in sha' Allah. if this isn't removed, no harm is done,
-      // since the page size has changed anyway, and so only the new images will be used.
-      // the page sets are currently not compatible (since one doesn't have ayah markers
-      // and one does).
-      final File newMadani = new File(baseDir, "new_madani");
```

Comment:
this used to delete the old new madani directory - it's not needed and it's been there for enough releases to hopefully have deleted those databases for people on the old version.